assets/src/edit-story/components/library/panes/media/mediaEditDialog.js

```diff
@@ -0,0 +1,211 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import { useCallback, useState } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useAPI } from '../../../../app/api';
+import Dialog from '../../../../components/dialog';
+import { Plain } from '../../../../components/button';
+import { useSnackbar } from '../../../../app/snackbar';
+import UseMedia from '../../../../app/media/useMedia';
+import useSmallestImage from './useSmallestImage';
+
+const styledMediaThumbnail = css`
+  display: flex;
+  width: 152px;
+  margin-right: 28px;
+`;
+
+const Image = styled.img`
+  ${styledMediaThumbnail}
+`;
+
+const Video = styled.video`
+  ${styledMediaThumbnail}
+`;
+
+const DialogBody = styled.div`
+  max-width: 480px;
+  display: flex;
+  justify-content: left;
+  align-items: flex-start;
+`;
+
+const MetadataTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const MediaTitleText = styled.div`
+  font-family: ${({ theme }) => theme.fonts.title.family};
+  line-height: ${({ theme }) => theme.fonts.title.lineHeight};
+  font-size: ${({ theme }) => theme.fonts.title.size};
+  font-weight: ${({ theme }) => theme.fonts.title.weight};
+  color: ${({ theme }) => theme.colors.bg.v11};
+`;
+
+const MediaSizeText = styled.div`
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  color: ${({ theme }) => theme.colors.bg.v11};
+`;
+
+const Input = styled.input`
+  background: ${({ theme }) => theme.colors.bg.v13};
+  border: 1px solid ${({ theme }) => theme.colors.fg.v3};
+  box-sizing: border-box;
+  border-radius: 4px;
+  font-family: ${({ theme }) => theme.fonts.input.family};
+  line-height: ${({ theme }) => theme.fonts.input.lineHeight};
+  font-size: ${({ theme }) => theme.fonts.input.size};
+  padding: 7px 10px;
+  margin-top: 20px;
+  margin-bottom: 4px;
+`;
+
+const DialogDescription = styled.div`
+  font-family: ${({ theme }) => theme.fonts.description.family};
+  line-height: ${({ theme }) => theme.fonts.description.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.description.weight};
+  font-size: ${({ theme }) => theme.fonts.description.size};
+  color: ${({ theme }) => theme.grayout};
+`;
+
+const imageDialogTitle = __('Edit Image', 'web-stories');
```

Comment:
You could use `sprintf` here. So something like this. 
```
/* translators: %s is file type. */
const dialogTitle = sprintf( __('Edit %s', 'web-stories'), type );
/* translators: %s is file type. */
const dialogDescription = sprintf( __(
  'Describe the appearance and function of a %s. Leave empty if the %s is purely decorative',
  'web-stories'
), type );
```
Makes this a little cleaner.

Reply:
Disagree on that one 🙂 

The sentences might need to be translated slightly different depending on whether it's an image or video (e.g. declension). So not using `sprintf` here is fine.

The second example (`Describe the appearance and function of a %s`) doesn't  work properly in English either:

`Describe the appearance and function of a video` would work
`Describe the appearance and function of a image` would be wrong (should be `an image`).